src/components/userFeedback/UserFeedback.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { extend } from 'underscore';
+
+export interface IUserFeedbackProps {
+  feedbackText: string;
+  state: string;
+  extraClasses?: string;
+  displayOnShow?: string;
```

Comment:
displayOnShow vs displayClass, what's the difference? I think displayOnShow should be renamed for displayClass to fit with the IUserFeedbackStyle interface.